From 683aacce5b642a3c735f8040601e0de345d028da Mon Sep 17 00:00:00 2001
From: Jan Henke <Jan.Henke@taujhe.de>
Date: Sat, 17 Mar 2018 19:47:52 +0100
Subject: [PATCH] Disable -WError when compiling tests

GCC 7 generates new warnings on some of the test files, which lead to
build failures. -WError generally is helpful while developing to enforce
a zero error policy, but not so much when packaging. Gentoo bug 644042
---
 tests/CMakeLists.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/tests/CMakeLists.txt b/tests/CMakeLists.txt
index 0e08d77..86e9af7 100644
--- a/tests/CMakeLists.txt
+++ b/tests/CMakeLists.txt
@@ -41,7 +41,6 @@ target_compile_options(gsl_tests_config INTERFACE
         -Wcast-align
         -Wconversion
         -Wctor-dtor-privacy
-        -Werror
         -Wextra
         -Wno-missing-braces
         -Wnon-virtual-dtor
@@ -129,7 +128,6 @@ target_compile_options(gsl_tests_config_noexcept INTERFACE
         -Wcast-align
         -Wconversion
         -Wctor-dtor-privacy
-        -Werror
         -Wextra
         -Wno-missing-braces
         -Wnon-virtual-dtor
-- 
2.16.1

